src/ngx_http_lua_log.c

```diff
@@ -275,6 +276,9 @@ ngx_http_lua_inject_log_api(lua_State *L)
     lua_pushcfunction(L, ngx_http_lua_ngx_log);
     lua_setfield(L, -2, "log");
 
+    lua_pushcfunction(L, ngx_http_lua_ngx_get_log);
```

Comment:
I'm sorry, I missed the news, I was ready to submit this evening, but I did not finish it. I will submit a new version tomorrow morning.